templates/layout.html

```diff
@@ -79,126 +79,34 @@
         </ul>
       </div>
 
-      <!-- Collect the nav links, forms, and other content for toggling -->
+      {{!-- Collect the nav links, forms, and other content for toggling --}}
       <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
         <ul class="nav navbar-nav navbar-right">
           {{#if currentUser}}
-            <li class="{{#if isActiveRoute 'newCompetition'}}active{{/if}} leaf">
-              <a href="{{pathFor 'newCompetition'}}"
-                 data-toggle="tooltip" data-placement="bottom" data-container="body"
-                 title="New competition">
-                <span class="glyphicon glyphicon-plus"></span>
-              </a>
-            </li>
+            {{#with newCompetitionTab}}
+              {{> oneTab}}
```

Comment:
That was my dream as well. But since the main use of `oneTab` is...

      {{#each managerTabs}}
        {{> oneTab}}
      {{/each}}

...we have to use `Template.parentData()` to get to things we need inside the template. And *that* means I have to use #with here to make `Template.parentData()` work properly here.

In the next version on Meteor there will be a `{{#each tab in managerTabs}}` construct that adds this.tab to the data context instead of replacing it. Then things like this could be less painful, and this would become 

    {{# let tab=newCompetitionTab}}
      {{> oneTab}}
    {{/let}}